src/engine-scripts/puppet/jsReady.js

```diff
@@ -0,0 +1,24 @@
+module.exports = async (page) => {
+	await page.emulateMediaFeatures([
+		{ name: 'prefers-reduced-motion', value: 'reduce' },
+	]);
+	await page.evaluate( () => {
+		window.dispatchEvent( new Event( 'resize' ) );
+		const skin = mw.config.get( 'skin' );
+		let moduleName;
+		switch( skin ) {
+			case 'vector-2022':
+				moduleName = 'skins.vector.js';
+				break;
+			case 'vector':
+				moduleName = 'skins.vector.legacy.js';
+				break;
+			default:
+				return true;
+		}
+		return mw.loader.getState( moduleName ) === 'ready';
+	});
+
+	// wait for animation frame or two.
+	await page.waitForTimeout(2000);
```

Comment:
```suggestion
	await page.waitForTimeout(1000);
```

Can this be lowered to 1 second? 2 seconds seems longer that our typical animation durations. With the tabs, I think the default for jquery's `.animate` is [400ms](https://api.jquery.com/animate/)